Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -113,6 +124,8 @@ private class TabCell: UITableViewCell {
 
         innerStroke.frame = background.frame
 
+        closeTab.frame = CGRect(x: backgroundHolder.frame.width - TabTrayControllerUX.CloseButtonSize/2 - TabTrayControllerUX.Margin, y: TabTrayControllerUX.CloseButtonMargin, width: TabTrayControllerUX.CloseButtonSize, height: TabTrayControllerUX.CloseButtonSize)
```

Comment:
I'm not sure I understand how you're calculating the x value. Why divide `TabTrayControllerUX.CloseButtonSize` by 2? And why subtract `TabTrayControllerUX.Margin`? The end result might look OK right now, but we should make sure the math makes sense instead of arbitrarily putting values together.

I think the following is more intuitive:
```
backgroundHolder.frame.width - TabTrayControllerUX.CloseButtonSize - TabTrayControllerUX.CloseButtonMargin
```